mainCode/sportwaremanager.py

```diff
@@ -13,19 +13,20 @@ def set_wears(self, wears):
     def get_wears(self):
         return self.wears
 
-    def sort_by_price(self, order):
+    # this function sort wears by  DESC and ASC price, if the set parameter order True it will  sort  by DESC
+    # that`s why the func is  named in  such way. But you can sort wears ASC if you set order like False
 
-        if order == Enums.SortType.ASC:
-            self.wears.sort(key=lambda c: c.get_price)
-        else:
-            self.wears.sort(key=lambda c: c.get_price(), reverse=True)
+    def sort_by_price_DESC(self, order):
```

Comment:
A function that can do both ASC and DESC sorting - should not have DESC as a part of its name, despite that fact it's described in the comment